collection-scripts/gather_host_network_logs

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+source $(dirname "$0")/common.sh
+get_log_collection_args
+. gather_service_logs_util
+
+BASE_COLLECTION_PATH="/must-gather"
+HOST_NETWORK_NODE_LOGS=$BASE_COLLECTION_PATH/host_network_logs
+
+mkdir -p "${HOST_NETWORK_NODE_LOGS}"/
+
+PIDS=()
+echo "INFO: Collecting host networking logs"
+
+collect_service_logs --role=master "on-prem-resolv-prepender"
+collect_service_logs --role=worker "on-prem-resolv-prepender"
```

Comment:
Gathering logs from the workers (when there are 500 nodes) might be a problem is there any way to limit this ?

Reply:
Are clusters with 500 nodes a real thing? From which perspective it may be a problem, are we now talking about the `time oc must-gather [...]` or about number of files collected?

Reply:
@mkowalski we need to watch for both (we don't want our collection tool to take too long to run or to collect GB's of data; that are impossible to review/process).